templates/takeovers/_kubernetes_webinar.html

```diff
@@ -0,0 +1,15 @@
+<section class="p-strip--accent is-dark is-deep p-takeover">
+  <div class="row">
+    <div class="u-equal-height u-vertically-center">
+      <div class="col-6">
+        <h1>Build your Kubernetes strategy with Ubuntu</h1>
+        <p>Learn how to get started.</p>
+        <div class="u-hidden--medium u-hidden--large u-padding-bottom--x-large u-align--center">
+          <img src="{{ ASSET_SERVER_URL }}331f3baf-kubernetes-logo.svg" alt="Kubernetes logo" />
+        </div>
+        <p><a class="p-button--neutral" href="http://ubunt.eu/GUWzfp" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Kubernetes webinar takeover', 'eventLabel' : 'Watch the webinar', 'eventValue' : undefined });">Watch the webinar</a></p>
+        <p class="intro"><a class="external" href="http://ubunt.eu/jMgdKN " onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Kubernetes takeover', 'eventLabel' : 'Find out more about Kubernetes', 'eventValue' : undefined });">Find out more about Kubernetes&nbsp;&rsaquo;</a></p>
```

Comment:
I'm going to remove the class. The shortlinker is an external URL but it resolves to an internal link.